CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 
 All notable changes to the Ansible VS Code extension will be documented in this file.
 
-[//]: # (DO-NOT-REMOVE-versioning-promise-START)
+[//]: # 'DO-NOT-REMOVE-versioning-promise-START'
```

Comment:
I didn't like the quotes. Just remove them. The braces are there to emphasize that this thing is a sort of an identifier. What _exactly_ the formatter does not like here? Maybe we'd just use the HTML comment syntax to mitigate the problem you may be seeing.

Reply:
Prettier does not like round braces, you don't like quotes, lets see if nothing should suite everyone :) -- testing.

Reply:
Apparently it works and is even nicer.

Reply:
It's not about the niceness, it's about being clear that this is not to be modified because it may break sphinx silently IIRC.